tests/unit/lms/security_test.py

```diff
@@ -23,7 +23,13 @@ class TestIncludeMe:
     def test_it_sets_security_policy(self, pyramid_config, SecurityPolicy):
         includeme(pyramid_config)
 
-        SecurityPolicy.assert_called_once_with()
+        SecurityPolicy.assert_called_once_with(
+            path_policy_mapping={
```

Comment:
Asserting the exact configuration.

Reply:
You could avoid having to do this if `path_policy_mapping` were hard-coded